src/pages/MyCakesList.js

```diff
@@ -15,30 +15,36 @@ export default function MyCakesList() {
     localStorage.setItem("recipesArray", JSON.stringify(listOfCakes));
   }, [listOfCakes]);
 
+  useEffect(() => {
+    localStorage.setItem("favoritesArray", JSON.stringify(listOfFavs));
+  }, [listOfFavs]);
+
   function handleDeleteItemFromList(id) {
     const updatedList = listOfCakes.filter((cake) => cake.id !== id);
     setListOfCakes(updatedList);
   }
-
+  // very slick solution from Adriana
```

Comment:
🤩 🦄 so cool! But you probably want to delete this comment before you merge.

Reply:
I will delete it. But this solution I "borrowed" from you and wanted to say thank you